google-cloud-bigtable/lib/google/cloud/bigtable/mutation_operations.rb

```diff
@@ -245,7 +245,7 @@ def check_and_mutate_row key, predicate, on_match: nil, otherwise: nil
           response = service.check_and_mutate_row(
             path,
             key,
-            predicate_filter: predicate.to_grpc,
+            predicate_filter: predicate&.to_grpc,
```

Comment:
It seemed important to be able to pass a null/no-op value here, and I couldn't figure out how to do it, except via something very hacky like

```ruby
foo = {}
def foo.to_grpc
  nil
end

table.check_and_mutate_row(key, foo, ...)
```

Reply:
Thanks. LGTM.